ainstein_radar_drivers/nodes/wayv_node

```diff
@@ -728,7 +617,8 @@ if __name__ == "__main__":
         time.sleep(2)  # delay long enough for the radar to connect over WiFi
 
         #Initialize signal watcher for config 
-        smokesignal.on('config_ready', wayv_intf.supervisor)
+        supervisor_thread = threading.Thread(target=wayv_intf.supervisor)
+        supervisor_thread.start()
```

Comment:
Maybe I am not understanding this here but it looks like you are running two instances of `supervisor()`? And the first instance waits two seconds after it is called on line 615 then there is a 2 second sleep timer and another thread of it is called? What is the point of this flow?